utils/_weblog.py

```diff
@@ -56,12 +57,21 @@ def __init__(self, data):
 
 
 class _Weblog:
-    _grpc_port = 7778
-
     def __init__(self):
-        if "DOCKER_HOST" in os.environ:
-            self.domain = os.environ["DOCKER_HOST"]
-            self.domain = self.domain.replace("ssh://docker@", "")
+        if "WEBLOG_PORT" in os.environ:
```

Comment:
I'm wondering if it would be a good idea to prefix those new env var with (`SYSTEM_TESTS_`) to prevent any collusion, and make crystal clear what is their usage when you see them in conf files, WDYT ?

Reply:
Indeed, probably worth a larger audit of env vars and prefix in a one shot refactor, so I feel it's out of scope for this PR.

Reply:
They're only used for passing information through `docker run` via `run_scenario` in `run.sh`, so they won't change.

Reply:
Fixed by 0cff8b78